documentation/4.0/content/release-notes.md

```diff
@@ -62,6 +63,14 @@ draft: false
 
 ### Change log
 
+#### Operator 4.0.4
+
+* Resolved an issue related to the `logHomeLayout` field during schema conversion from "v8" to "v9" Domains.
+  * Operator 4.0 introduced the `logHomeLayout` field to [control the organization of WebLogic Server instance log files](https://oracle.github.io/weblogic-kubernetes-operator/managing-domains/accessing-the-domain/logs/).
+  * The default value for `logHomeLayout` is `ByServers`; however, the Operator 3.4 behavior was consistent with the value of `Flat`.
```

Comment:
however, the Operator 3.4 -> however, the operator 3.4 
(On the home page, we have WebLogic Kubernetes Operator (the “operator”))... so the term should not be capitalized when used alone.)